Makefile

```diff
@@ -83,4 +84,6 @@ clean_mbedtls:
 clean_http_parser:
 	$(MAKE) -C $(HTTP_PARSER_PATH) clean
 
--include main.c.d
\ No newline at end of file
+pre-build:
+	git config core.hooksPath hooks
+-include main.c.d
```

Comment:
Use [String Substitution](https://www.gnu.org/software/make/manual/html_node/Text-Functions.html) to avoid hardcoded `main.c.d`.